src/Dijkstra/AST/Examples/RWS.agda

```diff
@@ -0,0 +1,51 @@
+module Dijkstra.AST.Examples.RWS where
+
+open import Data.Product      using (_×_ ; _,_)
+open import Dijkstra.AST.Core
+open import Function          using (case_of_)
+open import Haskell.Prelude
+open import Relation.Binary.PropositionalEquality
+
+module Example1 (A : Set) where
+
+  open import Data.Nat         renaming (ℕ to Nat) using (_+_ ; suc ; zero)
+  open import Dijkstra.AST.RWS A A (List A)
+  open        ASTPredTrans     RWSPT
+  open        RWSSyntax
+
+  sucn≡n+1 : ∀ (n : Nat) -> suc n ≡ n + 1
```

Comment:
I think this is just `+-comm` from the standard library.

Reply:
`+-comm` is a property about two Nats.   This property I defined is about one Nat.
I couldn't find anything in `Data.Nat.Properties` like it (to my surprise).

Reply:
Apologies for response in the form of a PR on a branch called `mark-tweak-rws-example-and-harold` :smile:.  #221